assets/src/js/product-editor.js

```diff
@@ -512,6 +512,16 @@
                         Dokan_Editor.loadSelect2();
                         Dokan_Editor.bindProductTagDropdown();
                         Dokan_Editor.attribute.reArrangeAttribute();
+
+                        if ( "variable" !== $( 'select#product_type' ).val() ) {
+                            let labels = $( 'div.dokan-product-attribute-wrapper label.show_if_variable' );
+
+                            for( let label of labels ) {
+                                if ( label.textContent.includes( "Used for variations" ) ) {
```

Comment:
Isn't there any obvious way to check this? What if the text content is translated. Kindly make it sure.